src/main/java/com/google/cloud/security/scanner/sources/LiveProjectSource.java

```diff
@@ -223,7 +223,10 @@ private boolean refreshProjects(String nextPageToken) throws IOException {
         if (project.getParent() != null) {
           orgId = project.getParent().getId();
         }
-        this.projects.add(new GCPProject(project.getProjectId(), orgId, project.getName()));
+        if (project.getLifecycleState() == null
```

Comment:
Just want to double-check here that the correct condition here is null, vs NOT null?